src/Humanizer/TimeUnitEnum.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Humanizer
+{
+    public enum TimeUnitEnum
```

Comment:
There is already a `TimeUnit` enum present in `Humanizer.Localisation` namespace, see [TimeUnit.cs](https://github.com/MehdiK/Humanizer/blob/master/src/Humanizer/Localisation/TimeUnit.cs)